modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceAddToCartFragment.testcase

```diff
@@ -1,7 +1,6 @@
 @component-name = "portal-commerce"
 definition {
 
-	property custom.properties = "jsonws.web.service.paths.excludes=${line.separator}company.security.strangers.verify=false${line.separator}feature.flag.COMMERCE-5898=true";
```

Comment:
@alessiorendina we need to keep these custom.properties, we just need to remove the feature flag part:
${line.separator}feature.flag.COMMERCE-5898=true